manufacturing/internals/tray/README.md

```diff
@@ -18,24 +18,14 @@ spot with new holes or cut-outs for alternate components being tested.
 In addition to supporting the main pneumatics tray, the equipment bracket also supports the display and filter panels
 on the front face of the ventilator. Those are detailed in other assembly pages.
 
-## Custom parts
-
-| Part  | Source | Export |
-|:------|:-------|---|
-| Equipment bracket | [.ipt](equipment_bracket.ipt) | **TODO** |
-| Pneumatics tray   | [.ipt](pneumatics_tray.ipt)     | **TODO** |
-
-### Fabrication Drawings
-
->**TODO: Make new drawings**
->
-Fabrication drawings.
+----
+## Assembly
```

Comment:
Also, assembly should definitely be at the bottom. It's ok that you moved the other stuff down, but I don't know why this would come first.